dist/protractor.conf.js

```diff
@@ -76,6 +77,17 @@ exports.config = {
   }],
 
   onPrepare: function () {
+    const generatedReportsDirectory = config.projectPath + config.reports + '/report';
+    const files = fs.readdirSync(generatedReportsDirectory).filter(file => {
```

Comment:
This variable is unnecessary, the same with a for "loop". ReaddirSync returns an array, so do filter, instead of doing for ... in . You could chain `.forEach` and run everything in simple 3-liner.

```
fs.readdirSync(generatedReportsDirectory)
  .filter(file => fs.statSync(path.join(generatedReportsDirectory, file)).isFile())
  .forEach(file => fs.unlinkSync(path.join(generatedReportsDirectory, file));
```